### PR TITLE
[Doc][Upgrade] Fix tiny error in upgrade docs

### LINKED ISSUE #10846: [Doc][Upgrade] Tiny error in upgrade docs
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

- Got some feedback from users in DS Slack Channel, `SPRING_DATASOURCE_URL` example value in `upgrade` docs is not quoted, which misleads users.

image

### Documentation Links

https://dolphinscheduler.apache.org/en-us/docs/latest/user_doc/guide/upgrade.html

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by EricGao888**
## Purpose of the pull request
*  Fix tiny error in upgrade docs.
* Got some feedback from users in DS Slack Channel, SPRING_DATASOURCE_URL example value in upgrade docs is not quoted, which misleads users.
* This PR closes: #10846 

## Brief change log

* Quote example datasource url value in upgrade docs.

## Verify this pull request

* Verified by manual test.

### CODE REVIEW

File: docs/docs/en/guide/upgrade.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ jar package and add it to the `./tools/libs` directory, then change `./bin/ env/
     ```shell
     export DATABASE=${DATABASE:-mysql}
     export SPRING_PROFILES_ACTIVE=${DATABASE}
-    export SPRING_DATASOURCE_URL=jdbc:mysql://127.0.0.1:3306/dolphinscheduler?useUnicode=true&characterEncoding=UTF-8
+    export SPRING_DATASOURCE_URL="jdbc:mysql://127.0.0.1:3306/dolphinscheduler?useUnicode=true&characterEncoding=UTF-8"
```
Comment (zhongjiajie) [inline docs/docs/en/guide/upgrade.md]: It seems using `&useSSL=false` in the connection param is necessary too. I have to add it to connect mysql server 5.7

Comment (zhongjiajie) [inline docs/docs/en/guide/upgrade.md]: It seems using `&useSSL=false` in the connection param is necessary too. I have to add it to connect mysql server 5.7

Comment (EricGao888) [reply docs/docs/en/guide/upgrade.md]: Sure, thx for pointing this out. I will add it in the next commit.

Comment (EricGao888) [reply docs/docs/en/guide/upgrade.md]: Will fix it

**Review by SbloodyS — APPROVED** [review]
LGTM

**Review by davidzollo — APPROVED** [review]
LGTM

**Pull request merged**